src/common/config.rs

```diff
@@ -26,6 +74,7 @@ pub struct WatchtowerConfig {
 pub struct StakeholderConfig {
     pub xpub: bip32::ExtendedPubKey,
     pub watchtowers: Vec<WatchtowerConfig>,
+    pub emergency_address: Address,
```

Comment:
nit: Change address for revault_tx::EmergencyAddress using a serde Deserializer
example: https://github.com/re-vault/revault-gui/blob/master/src/revaultd/model.rs#L131

Reply:
Added a commit upstream for this, as we may use it in other projects too.  See https://github.com/re-vault/revault_tx/pull/58#issuecomment-781419432